test_data/makedata_suites/021_foxx_utf-8.js

```diff
@@ -0,0 +1,150 @@
+/* global print, fs, db, internal, arango, assertTrue */
+/* global loadFoxxIntoZip, installFoxx, deleteFoxx, itzpapalotlZip, minimalWorkingZip, assertEqual */
+
+
+(function () {
+  let extendedNames = ["ᇤ፼ᢟ⚥㑸ন", "に楽しい新習慣", "うっとりとろける", "זַרקוֹר", "ስፖትላይት", "بقعة ضوء", "ուշադրության կենտրոնում", "🌸🌲🌵 🍃💔"];
+  let aardvarkRoute = '/_db/_system/_admin/aardvark/index.html';
+  let shouldValidateFoxx;
+  const onlyJson = {
+    'accept': 'application/json',
+    'accept-content-type': 'application/json'
+  };
+  let testFoxxRoutingReady = function() {
+    for (let i = 0; i < 200; i++) {
+      try {
+        let reply = arango.GET_RAW('/this_route_is_not_here', onlyJson);
+        if (reply.code === 404) {
+          print("selfHeal was already executed - Foxx is ready!");
+          return 0;
+        }
+        print(" Not yet ready, retrying: " + reply.parsedBody);
+      } catch (e) {
+        print(" Caught - need to retry. " + JSON.stringify(e));
+      }
+      internal.sleep(3);
+    }
+    throw new Error("foxx routeing not ready on time!");
+  };
+  let testFoxxReady = function(route) {
+    for (let i = 0; i < 200; i++) {
+      try {
+        let reply = arango.GET_RAW(route, onlyJson);
+        if (reply.code === 200) {
+          print(route + " OK");
+          return 0;
+        }
+        let msg = JSON.stringify(reply);
+        if (reply.hasOwnProperty('parsedBody')) {
+          msg = " '" + reply.parsedBody.errorNum + "' - " + reply.parsedBody.errorMessage;
+        }
+        print(route + " Not yet ready, retrying: " + msg);
+      } catch (e) {
+        print(route + " Caught - need to retry. " + JSON.stringify(e));
+      }
+      internal.sleep(3);
+    }
+    throw new Error("foxx route '" + route + "' not ready on time!");
+  };    
+  return {
+    isSupported: function (version, oldVersion, options, enterprise, cluster) {
+      return options.testFoxx;
+    },
+    makeDataDB: function (options, isCluster, isEnterprise, database, dbCount) {
+      let dbName = `${extendedNames[0]}FoxxTest${extendedNames[3]}_${dbCount}`;
```

Comment:
Why we use only 2 items from extendedNames array?

Reply:
to have a rough overview.